source/blog/2021-04-19-dbal-2.13.1-3.1.0.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "New Release: Doctrine DBAL 2.13.1 and 3.1.0 with important Forward Compatibility fix"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2021/04/19/dbal-2.13.1-3.1.0.html
+---
+
+Last month [we released DBAL
+2.13.0](https://www.doctrine-project.org/2021/03/29/dbal-2.13.html) as an
+important push for the ecosystem towards DBAL 3 with an extensive deprecation
+layer and forwards compatibility.
+
+We made a few mistakes, given that the forward compatibility layer is quite complex.
+As such we have now released Doctrine DBAL 2.13.1 and 3.1.0 with two new APIs
+that improve the forward compatiblity.
+
+The problem lies in `Statement::execute`: 2.13.0 would return a `bool` and 3.0.0
+would return a `Result` from this method. This is a BC break that we cannot fix
```

Comment:
It's not that we're trying to fix a BC break, we're trying to provide an upgrade path (via the FC layer).